pages/about.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Link from 'next/link'
+
+function about() {
+  return (
+    <div className='text-3xl undeline'>
```

Comment:
Look out for these typos(misspelled classes). They might ruin your whole day unknowingly 🥲.